pynicotine/slskmessages.py

```diff
@@ -1700,15 +1700,15 @@ def parseNetworkMessage(self, message):
 
     def makeNetworkMessage(self):
         if self.pic is not None:
-            pic = chr(1) + self.packObject(self.pic)
+            pic = bytes(1) + self.packObject(self.pic)
         else:
-            pic = chr(0)
+            pic = bytes(0)
 
         return (self.packObject(self.descr) +
                 pic +
                 self.packObject(NetworkIntType(self.totalupl)) +
                 self.packObject(NetworkIntType(self.queuesize)) +
-                chr(self.slotsavail) +
+                self.slotsavail.encode() +
```

Comment:
Not sure about this, but I think it's not safe to use either, better to use `bytes([self.slotsavail])` as above